src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -16,6 +16,10 @@ public sealed class XamlLoadPermission : CodeAccessPermission, IUnrestrictedPerm
         public XamlLoadPermission(PermissionState state) { }
         public XamlLoadPermission(XamlAccessLevel allowedAccess) { }
         public XamlLoadPermission(IEnumerable<XamlAccessLevel> allowedAccess) { }
+        [ComVisible(false)]
+        public override bool Equals(object obj) { return false; }
```

Comment:
This is not a valid implementation because `x.Equals(x)` returns false. You should use `ReferenceEquals(this, obj)` or throw an exception if you don't want to support equality.

Reply:
This impl. will get deleted from dotnet/wpf shortly, and we'll consume the one from https://github.com/dotnet/corefx/pull/38405. Let's make sure we get the corefx PR right.